NEWS.md

```diff
@@ -1,6 +1,34 @@
 
 # tidyselect 0.1.1.9000
 
+* The semantics for data expressions have been changed back to the old
+  behaviour because the new rules were causing a lot of trouble. From
+  now on data expressions (symbols and calls to `:` and `c()`) can
+  refer to both registered variables and to objects from the context.
+
+  However the semantics for context expressions (any calls other than
```

Comment:
I thought we were changing this too.  What's the advantage of having special behaviour for `:` and `c`? Can't we tell people to unquote if they want to use variables in the overscope?